ui/imports/utils/Utils.qml

```diff
@@ -13,7 +13,8 @@ QtObject {
     }
 
     function isChatKey(value) {
-        return startsWith0x(value) && isHex(value) && value.length === 132
+        return (startsWith0x(value) && isHex(value) && value.length === 132) || 
+               /^z[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{48}$/.test(inputValue)
```

Comment:
Could we use `globalUtilsInst.isChatKey(value)` (needs to be introduced, should use `isCompressedPubKey` from `conversion.nim`) instead? That way we would avoid logic duplication and possible errors.